game/views/user.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from pyQAtest_tamagochi.game.views.interface import GameInterface
+from pyQAtest_tamagochi.game.models import Tamagochi, random, pets
+
+
+class User(GameInterface):
```

Comment:
Propose to implement cli interpreter for not to simulate users actions

Reply:
done